docs/UserGuide.md

```diff
@@ -326,36 +345,53 @@ Lists all the entries in the user's income list.
 
 **Format:** `list incomes`
 
-The user will then be prompted to select a time range. 
+The user will then be prompted to select a time range from the following options:
+1. All
+2. This week
+3. Last week and this week
+4. This month
+
+Only entries that are dated within the selected time range will be displayed.
+<br> E.g. If today is 20 November 2024 (Wednesday) and the user chooses option 2 (this week),  only income entries
+from the current week's Monday (18 November 2024) to Sunday (24 November 2024) will be shown.
+
 Only entries that are within the time range will be displayed.
 
 **Example Input and Output**
 
 **Input:** `list incomes`
 ```
 	____________________________________________________________
-	Select time range:
+	List income entries for:
 	[1] All
-	[2] Weekly
-	[3] Biweekly
-	[4] Monthly 
-
+	[2] This week
```

Comment:
should we mention explicitly how the 2 weeks work

Reply:
In the output? or in the ug

Reply:
Okay, maybe I'll explain for all options

Reply:
done